composer.json

```diff
@@ -11,12 +11,12 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "phpseclib/phpseclib": "^2.0",
```

Comment:
I don't really like having these dependencies here though. I want this library to be used in any application and this might conflict with application dependencies.

Maybe it's best to move these back to `dev` and add a suggests part + mention it in the readme that if people want to use Energon they have to include these dependencies.